src/python/doc/cubical_complex_tflow_itf_ref.rst

```diff
@@ -0,0 +1,40 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for cubical persistence
+########################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from cubical persistence
+-----------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import CubicalLayer
+    import tensorflow as tf
+
+    X = tf.Variable([[0.,2.,2.],[2.,2.,2.],[2.,2.,1.]], dtype=tf.float32, trainable=True)
+    cl = CubicalLayer(dimension=0)
```

Comment:
The doc says `dimensions=list`, you may have forgotten to update it. I am surprised the testsuite didn't notice it, I thought it executed the testcode in the doc...

Reply:
Good catch, thanks. I fixed it.